Components/BlockPostHeader/Partials/Image/_style.scss

```diff
@@ -0,0 +1,12 @@
+.image {
+  display: block;
+  height: auto;
+  margin: 0;
+  width: 100%;
```

Comment:
I think you don't need all of the above. Instead though, please use the same markup and styling as in `BlockImage` (with <figure>).

Reply:
Integrated BlockImage ... but generally there is no CSS at all related to the image or structures shown in the lazysizes docs.